packages/@dcl/ecs/src/systems/events.ts

```diff
@@ -0,0 +1,155 @@
+import { InputAction } from '../components/generated/pb/decentraland/sdk/components/common/input_action.gen'
+import { PBPointerEventsResult_PointerCommand } from '../components/generated/pb/decentraland/sdk/components/pointer_events_result.gen'
+import { PointerEventType } from '../components/generated/pb/decentraland/sdk/components/pointer_hover_feedback.gen'
+import { Entity, IEngine, IInput } from '../engine'
+
+declare const engine: IEngine
+
+export namespace EventsSystem {
+  export type Callback = (
+    event: PBPointerEventsResult_PointerCommand
+  ) => void | Promise<void>
+
+  export type Opts = {
+    button: InputAction
+    hoverText?: string
+  }
+  type EventMapType = Map<PointerEventType, { cb: Callback; opts: Opts }>
+
+  const defaultOpts: Opts = {
```

Comment:
```suggestion
  const getDefaultOptions: Opts = () => ({
    button: InputAction.IA_ANY,
    hoverText: ''
  })
```
I think it's not the case right now, but if we could edit hoverFeedback or the button, it'll modify all the events with reference to the default.